loops-and-arrays.js

```diff
@@ -37,6 +45,10 @@ const numbers = [2, 4, 5, 29, 38];
 
 const squares = [30, 2, 8, 24, 11];
 
+for (let i = 0; i < 5; i++){
+    console.log(squares[i]*squares[i]*squares[i])
+}
+
 // Verwachtte uitkomsten:
 // Vóór het script zie je de originele waardes:
 // console.log(numbers) geeft: [30, 2, 8, 24, 11];
```

Comment:
met Math.pow kun je machtsverheffen:

const squares = [30, 2, 8, 24, 11];

for (let i = 0; i < squares.length; i++) {
    console.log(Math.pow(squares[i],3));
}